test/models/wager_test.exs

```diff
@@ -3,7 +3,7 @@ defmodule MovieWagerApi.WagerTest do
 
   alias MovieWagerApi.Wager
 
-  @valid_attrs %{amount: 42, place: 42}
+  @valid_attrs %{amount: 42, place: 42, user_id: 50, movie_round_id: 55}
```

Comment:
hm, what error message are you seeing? Adding `changeset = Wager.changeset(%Wager{}, params_with_assocs(:wager))` to the first test in the wager model doesn't return an error for me 🤔